### PR TITLE
URL: test IDNA ignored code points in host

### PR DESCRIPTION
**Pull request by rmisev**
See https://github.com/whatwg/url/pull/497 for context.

### CODE REVIEW

File: url/resources/toascii.json [hunk 1]
```diff
@@ -145,5 +145,27 @@
...
+    "comment": "IDNA ignored code points",
+    "input": "a\u00ADb",
+    "output": "ab"
+  },
+  {
+    "input": "a%C2%ADb",
+    "output": "ab"
+  },
+  {
+    "comment": "Empty host after the domain to ASCII",
```
Comment (annevk) [inline url/resources/toascii.json:159]:
```suggestion
    "comment": "Empty host after domain to ASCII",
```

**Review by annevk — APPROVED** [review]
These seem good to me, but would be good to run them in whatwg-url before landing. That usually catches something.

**Review by domenic — APPROVED** [review]
Confirmed that these tests fail in whatwg-url before the patch, and pass afterward; very nice!

**Review by Unknow-touch — COMMENTED** [review]
Ufkfkfki

**Pull request merged**